src/public/modules/forms/admin/directives/view-feedback.client.directive.js

```diff
@@ -1,10 +1,12 @@
 'use strict'
 
+const FormFeedback = require('../../../../services/FormFeedbackService')
```

Comment:
tiny nit but can rename this `FormFeedbackService` for consistency with other new services

Reply:
thanks! edited, and also for `feedback-form.client.component.js`